specs/device/administrators.yaml

```diff
@@ -0,0 +1,907 @@
+            type: enum
+            profiles:
+            - xpath:
+              - panorama-admin
+            validators:
+            - type: values
+              spec:
+                values:
+                - 'yes'
+            spec:
+              values:
+              - value: 'yes'
+            description: ''
+            required: false
+          - name: superreader
+            type: enum
+            profiles:
+            - xpath:
+              - superreader
+            validators:
+            - type: values
+              spec:
+                values:
+                - 'yes'
+            spec:
+              values:
+              - value: 'yes'
+            description: ''
+            required: false
+          - name: superuser
+            type: enum
+            profiles:
+            - xpath:
+              - superuser
+            validators:
+            - type: values
+              spec:
+                values:
+                - 'yes'
+            spec:
+              values:
+              - value: 'yes'
+            description: ''
+            required: false
+          - name: vsysadmin
+            type: list
+            profiles:
+            - xpath:
+              - vsysadmin
+              - entry
+              type: entry
+            validators: []
+            spec:
+              type: object
+              items:
+                type: object
+                spec:
+                  params:
+                  - name: vsys
+                    type: list
+                    profiles:
+                    - xpath:
+                      - vsys
+                      type: member
+                    validators: []
+                    spec:
+                      type: string
+                      items:
+                        type: string
+                    description: ''
+                    required: false
+                  variants: []
+            description: ''
+            required: false
+          - name: vsysreader
+            type: list
+            profiles:
+            - xpath:
+              - vsysreader
+              - entry
+              type: entry
+            validators: []
+            spec:
+              type: object
+              items:
+                type: object
+                spec:
+                  params:
+                  - name: vsys
+                    type: list
+                    profiles:
+                    - xpath:
+                      - vsys
+                      type: member
+                    validators: []
+                    spec:
+                      type: string
+                      items:
+                        type: string
+                    description: ''
+                    required: false
+                  variants: []
+            description: ''
+            required: false
+        description: ''
+        required: false
+    description: ''
+    required: false
+  - name: phash
```

Comment:
@kklimonda-cl so we store the computed phash that comes back from the server. How do we specify the password?